x-pack/plugins/index_lifecycle_management/public/sections/wizard/form_errors.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+
+import React, { cloneElement, Children, Fragment } from 'react';
+import { EuiFormRow } from '@elastic/eui';
+
+export const ErrableFormRow = ({
+  errorKey,
+  isShowingErrors,
+  errors,
+  children,
+  ...rest
+}) => {
+  return (
+    <EuiFormRow
+      isInvalid={
```

Comment:
`isShowingErrors` seems redundant in this component. We can simplify this component by passing in an empty array or just undefined for `errors` (and having it default to an empty array) whenever the consumer doesn't want to show errors.

Reply:
Additionally, we don't need to pass in both `errorKey` and `errors`. The consumer should be responsible for providing a simple array of errors:

```js
          <ErrableFormRow
            label="Choose where to allocate indices by node attribute"
            errors={isShowingErrors ? errors[STRUCTURE_NODE_ATTRS] || []}
          >
```